src/main/kotlin/com/figure/gradle/semver/internal/git/Git.kt

```diff
@@ -207,11 +202,14 @@ private fun Git.findYoungestTagCommitOnBranch(
         .firstOrNull { tags.containsKey(it.toObjectId()) }
 }
 
-internal fun Git.hasBranch(branch: GitRef.Branch): Boolean {
-    return runCatching {
-        branchList().setContains(branch.name).call()
-            .any { it.name.shortName().getOrThrow() == branch.name }
```

Comment:
For some reason this stopped working.. We should be checking all branches anyways.